libsolidity/ast/ASTAnnotations.h

```diff
@@ -256,6 +256,9 @@ struct IdentifierPathAnnotation: ASTAnnotation
 	Declaration const* referencedDeclaration = nullptr;
```

Comment:
Is this duplicated in the vector?

Reply:
Yes. I wanted to avoid changes all over the project, but I can only use the vector or even add a small function to access the last element